vite.config.ts

```diff
@@ -28,7 +28,9 @@ export default defineConfig({
       external: [
         "react",
         "react-dom",
+        "react-dom/server",
         "react/jsx-runtime",
+        "react/jsx-dev-runtime",
```

Comment:
Why? I understand react dom. But not sure I have the rationale behind this one? Is that something that usually gets injected by webpack dev server or similar?